module/assets/helper.js

```diff
@@ -0,0 +1,1026 @@
+// Dependencies
+const programsHelper = require(MODULES_BASE_PATH + "/programs/helper");
+const solutionsHelper = require(MODULES_BASE_PATH + "/solutions/helper");
+const userExtensionsHelper = require(MODULES_BASE_PATH +
+  "/user-extension/helper");
+const userRolesHelper = require(MODULES_BASE_PATH + "/user-roles/helper");
+const kafkaProducersHelper = require(ROOT_PATH + "/generics/kafka/producers");
+const telemetryEventOnOff = process.env.TELEMETRY_ON_OFF;
+
+module.exports = class AssetsHelper {
```

Comment:
@praveenKDass Fix method name and response of functions, improve variable names. Add function signature at every places.